### PR TITLE
PXP-444: [CLI] Support --verbose flag and WUKONG_LOG flag

### PR DESCRIPTION
**Pull request by jk-gan**
## Summary

The default log level is ERROR and can be overridden via `vebosity` flag and `WUKONG_LOG` env variable (with `off`, `error`, `warn`, `info`, `debug`, `trace` value).
```bash
# silences output
wukong <command> -q
# show warnings
wukong <command> -v
# show info
wukong <command> -vv
# show debug
wukong <command> -vvv
# show trace
wukong <command> -vvvv
```

Screenshot 2023-01-05 at 2 07 47 PM

Ticket: [PXP-444](https://mindvalley.atlassian.net/browse/PXP-444)

### Added
- [x] add logger

### Changed
- [x] the error output will be displayed via Error log 

[PXP-444]: https://mindvalley.atlassian.net/browse/PXP-444?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[PXP-444]: https://mindvalley.atlassian.net/browse/PXP-444?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/51#" title="Author self-approved">jk-gan</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by onimsha — APPROVED** [review]

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/51#" title="Author self-approved">jk-gan</a>*, *<a href="https://github.com/mindvalley/wukong-cli/pull/51#pullrequestreview-1240045253" title="Approved">onimsha</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
